helm_deploy/values-dev.yaml

```diff
@@ -10,7 +10,7 @@ image:
 
 ingress:
   enabled: true
-  enable_whitelist: false
+  enable_allowlist: false
```

Comment:
should this be `enable_allow_list`?  Same applies elsewhere...

Reply:
fixed this